style.css

```diff
@@ -182,18 +182,19 @@ img {
 }
 
 /* About section  */
-.about{
+.about {
   display:flex;
   align-items: center;
 }
 
-.about-paragraph{
+.about .paragraph div {
   text-align: center;
-  margin-right: 2em;
+  /* Pad based on window size */
+  padding: 0 calc(20px + 1.5vw);
 }
 
-.about p {
-  padding: .5rem;
+.about .paragraph div:nth-child(2) {
+    margin-top: 1rem;
```

Comment:
Cool hacky code that makes it so that the `div` underneath the other has a bit of spacing.